radicle/src/profile.rs

```diff
@@ -84,6 +84,11 @@ impl Profile {
 
     pub fn load() -> Result<Self, Error> {
         let home = self::home()?;
+
+        Self::load_at(home)
+    }
+
+    pub fn load_at(home: Home) -> Result<Self, Error> {
```

Comment:
Can be reverted

Reply:
reverted in https://github.com/radicle-dev/heartwood/pull/422/commits/75463a7872255a89490b362f7e11962c56cd15f2#diff-f5911c8765b18e1a9d9d0957001f22ff11cc9c0f5749f9293fae412ccd681362R56